structures/bid.js

```diff
@@ -48,10 +48,10 @@ exports.bid = function(config = null, auction = null) {
       lang: s.optional(s.string()),
       forwarder: s.optional(s.string()),
       decision_maker: s.optional(s.string()),
-      loaded: s.optional(s.boolean()),
+      loaded: s.optional(s.enums(["Yes", "No"])),
       tracker: s.optional(s.string()),
       driver: s.optional(s.string()),
-      score: s.optional(s.array(s.number())),
+      score: s.optional(s.array(s.string())),
```

Comment:
Breaking change... What about an array of string OR number ?

Reply:
Increase major version of tag ?

Reply:
No